index.md

```diff
@@ -12,7 +12,9 @@ This is an overview of all Android versions and their corresponding identifiers
 
 <table class="full-width">
   <tr>
-    <th>Version</th>
+    <th colspan="2">
```

Comment:
I would move the API Level column first, after all that's the "key" in this table and even your website is named after it. Everything else is explaining/detailing API levels.

So: API Level, Version Code, Codename, Version, Codename

(not sure about Codename's location, either of those two)